tutorials/haskell/snap-web-app/tutorial.md

```diff
@@ -0,0 +1,490 @@
+---
+title: How to create a web application in Haskell with Snap
+published: 2017-01-30
+ghc: 8.0.1
+lts: 7.18
+libraries: snap-core-1.0.1.0 snap-server-1.0.1.1 heist-1.0.1.0 HDBC-sqlite3-2.3.3.1    
+language: haskell
+description: In this tutorial we show you how to build a simple TODO web app using Snap framework
+author-name: Omar García
+---
+
+## Introduction
+
```

Comment:
We should avoid using the word "simple" to describe anything we are trying to teach.  If it doesn't seem simple to the reader, they could feel they are being talked down to.